src/service/video-manager-impl.js

```diff
@@ -405,6 +478,30 @@ class VideoEntry {
       mutate,
     });
   }
+
+
+  /**
+   * Returns whether the video is paused or playing after the user interacted
+   * with it or playing through autoplay
+   * @return {!../video-interface.VideoInterface} PlayingStates
+   */
+  getPlayingState() {
+    if (this.userInteracted_ && this.isPlaying_) {
```

Comment:
Currently video-interface is just an interface, if it were a base class, `play()` would be the right place for it. We make make a base class for video-interface at some point.

`userInteracted` flag is sort of misleading, it is only true in the case of autoplay, without autoplay we have no idea if user has interacted with the video (since most videos are iframes and we don't get `click` events on iframes, in the case of autoplay we put a div overlay on top of iframe to capture the click but in normal cases we can't do that)